lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -143,16 +156,9 @@ def is_intermediate_ca_set(self, mount: str, certificate: str) -> bool:
         intermediate_ca = self._client.secrets.pki.read_ca_certificate(mount_point=mount)
         return intermediate_ca == certificate
 
-    def is_intermediate_ca_set_with_common_name(self, mount: str, common_name: str) -> bool:
-        """Check if the intermediate CA is set for the PKI backend."""
-        intermediate_ca = self._client.secrets.pki.read_ca_certificate(mount_point=mount)
-        if not intermediate_ca:
-            return False
-        loaded_certificate = x509.load_pem_x509_certificate(intermediate_ca.encode("utf-8"))
-        existing_common_name = loaded_certificate.subject.get_attributes_for_oid(
-            x509.oid.NameOID.COMMON_NAME
-        )[0].value
-        return existing_common_name == common_name
+    def get_intermediate_ca(self, mount: str) -> str:
+        """Get the intermediate CA for the PKI backend."""
+        return self._client.secrets.pki.read_ca_certificate(mount_point=mount)
 
     def generate_pki_intermediate_ca_csr(self, mount: str, common_name: str) -> str:
```

Comment:
I know this was not introduced in this PR, but logs in this function seems a bit off.
Shouldn't it be something along the line: "Generated a CSR for the intermediate CA for the PKI backend"

Reply:
Done